cmd/allocator/main.go

```diff
@@ -87,6 +88,11 @@ const (
 	allocationBatchWaitTime          = "allocation-batch-wait-time"
 	readinessShutdownDuration        = "readiness-shutdown-duration"
 	httpUnallocatedStatusCode        = "http-unallocated-status-code"
+	processorGRPCAddress             = "processor-grpc-address"
+	processorGRPCPort                = "processor-grpc-port"
+	processorMaxBatchSize            = "processor-max-batch-size"
+	processorAllocationTimeout       = "processor-allocation-timeout"
+	processorReconnectInterval       = "processor-reconnect-interval"
```

Comment:
I'm wondering if we should just hardcode `processorAllocationTimeout` and `processorReconnectInterval` to values that work best 🤔 

Do we see there being value in making them configurable?